docs/source/Support/bskReleaseNotes.rst

```diff
@@ -73,6 +73,8 @@ Version |release|
   error. Rather, the process return code (0 for success, and anything else for failure) indicates the success.
 - The ``MAX_N_CSS_MEAS`` define is increased to 32 matching the maximum number of coarse sun sensors.
 - mixed bug in time to nano-seconds conversions in ``macros.py`` support file
+- Updated :ref:`thrusterPlatformReference` to add an input and output thruster config msg, and integral feedback term
+which dumps steady-state momentum in case of uncertainties on the CM location.
```

Comment:
The 2nd line must have two empty spaces, or it will not be added to the prior bullet.  The text must line up here with RST files.

Reply:
Done